src/lib.rs

```diff
@@ -15,19 +15,19 @@
 //! use std::thread;
 //! use std::time::Duration;
 //! use cast::*;
-//! 
+//!
```

Comment:
My editor trims these lines by default. Sorry if this is noise... I can remove it if so.